cmd/aro/rp.go

```diff
@@ -93,7 +94,7 @@ func rp(ctx context.Context, log *logrus.Entry) error {
 		return err
 	}
 
-	f, err := frontend.NewFrontend(ctx, log.WithField("component", "frontend"), _env, db, api.APIs, m, feCipher, kubeactions.New, features.NewResourcesClient)
+	f, err := frontend.NewFrontend(ctx, log.WithField("component", "frontend"), _env, db, api.APIs, m, feCipher, kubeactions.New, features.NewResourcesClient, compute.NewVirtualMachinesClient)
```

Comment:
This is getting out of hand :/ We need to think how to handle these enormous param lines.

Either `frontendOpts` or move what we can inside.

Reply:
#546